src/io/KeyMap.cpp

```diff
@@ -0,0 +1,47 @@
+#include <set>
+#include <iostream>
+
+#include "io/KeyMap.hpp"
+
+namespace chip8::io
+{
+    KeyMap::KeyMap()
+    {
+        //Should this be an assert instead
```

Comment:
I think that an assert here should be sufficient too. The KeyMap should be created only at the beginning, changing the mapping won't run this, so is not run-time/user related. As long as this is fine in our debug builds, it should work. Right?

Reply:
Yep, but I was not sure if tests on travis are built in release or debug mode.

Reply:
I'll set them to debug. 
`-DCMAKE_BUILD_TYPE=Debug`
should be sufficient, right?